cabal-testsuite/PackageTests/AutogenModulesToggling/Setup.hs

```diff
@@ -1,3 +1,4 @@
+{-# LANGUAGE OverloadedStrings #-}
```

Comment:
Not a bit deal but I guess this is a leftover?

Reply:
These are necessary because of `hookedPreProcessors` now receiving a `Suffix` rather than a string, which is part of #9484.